StartEycaActivationOrchestrator/index.ts

```diff
@@ -71,11 +76,24 @@ export const handler = function*(
       trackExAndThrow(e, "eyca.activate.exception.decode.activityOutput")
     );
 
-    if (updateEycaResult.kind !== "SUCCESS") {
+    if (updateEycaResult.kind !== "SUCCESS_VAL") {
       trackExAndThrow(
         new Error("Cannot activate EYCA Card"),
         "eyca.activate.exception.failure.activityOutput"
       );
+    } else {
```

Comment:
Since this branch is executed only if previous step doesn't throw, we can safely remove `else`

Reply:
That's true, but without the `else`, `updateEycaResult` is not visible as `ActivityResultSuccessWithValue` and I need to make a cast. What do you prefer?
![image](https://user-images.githubusercontent.com/75862507/109011538-4989f900-76b1-11eb-8e53-d3f8f0315383.png)

Reply:
You can try something like:
```
const updateEycaSuccessResult = ActivityResultSuccessWithValue.decode(
      updateEycaResult
    ).getOrElseL(() =>
      trackExAndThrow(
        new Error("Cannot activate EYCA Card"),
        "eyca.activate.exception.failure.activityOutput"
      )
    );

    const storedEycaCard = updateEycaSuccessResult.value;
```.

Reply:
I prefer the `else`, but sure I can

Reply:
Done in [#9d07ba5](https://github.com/pagopa/io-functions-cgn/pull/27/commits/9d07ba561352276df16d81ca4859d3d33e9c88fa)